build.gradle

```diff
@@ -111,9 +111,12 @@ task smoke(type: Test) {
 }
 
 task highLevelDataSetup(type: JavaExec) {
-  mainClass.set("uk.gov.hmcts.reform.adoption.common.ccd.HighLevelDataSetupApp")
-  classpath += sourceSets.main.runtimeClasspath
-  jvmArgs = ['--add-opens=java.base/java.lang.reflect=ALL-UNNAMED']
+  println("XXXXXXXXXXX:"+System.getenv("ENVIRONMENT"))
+  if (System.getenv("ENVIRONMENT") != "prod") {
+    mainClass.set("uk.gov.hmcts.reform.adoption.common.ccd.HighLevelDataSetupApp")
+    classpath += sourceSets.main.runtimeClasspath
+    jvmArgs = ['--add-opens=java.base/java.lang.reflect=ALL-UNNAMED']
+  }
```

Comment:
Could you please add else condition and a log saying `high level data setup is disabled in prod temporarily`